README.md

```diff
@@ -1,6 +1,89 @@
 # safe-network-launch-tool
 A cross platform tool to easily launch a SAFE Network test section from home
 
+## Installing the SAFE Vault
+
+A local SAFE network is bootstrapped by running several [SAFE vaults](https://github.com/maidsafe/safe_vault) which automatically interconnect forming a network.
+
+In order to run your own local network you'd need to follow these steps:
+- Download latest release from [safe_vault releases](https://github.com/maidsafe/safe_vault/releases/tag/0.20.1)
+- Untar/unzip the downloaded file into a directory of your choice
+- Execute this tool specifying the path of the `safe_vault` executable
+
+The following is an example of how to perform this on Linux or Mac:
+```shell
+$ mkdir ~/my-local-network
+$ cd ~/my-local-network
+$ curl -O https://github.com/maidsafe/safe_vault/releases/download/0.21.0/safe_vault-0.21.0-x86_64-unknown-linux-musl.tar.gz
+$ tar -xzvf safe_vault-0.21.0-x86_64-unknown-linux-musl.tar.gz
+```
+
+## Run a local network
+
+At current state of the [SAFE project](), a single-section SAFE network can be launched locally in our system. If the SAFE vault binary was downloaded and extracted at `~/my-local-network/` as described above, we can now launch the network using this tool following these steps:
+```shell
+$ git clone https://github.com/maidsafe/safe-network-launch-tool
+$ cd safe-network-launch-tool
+$ cargo run -- -p ~/my-local-network/safe_vault -v
+Launching with vault executable from: ~/my-local-network/safe_vault
+Network size: 8 vaults
+Launching genesis vault (#1)...
+Genesis vault contact info: ["127.0.0.1:59303"]
+Launching vault #2...
+Launching vault #3...
+Launching vault #4...
+Launching vault #5...
+Launching vault #6...
+Launching vault #7...
+Launching vault #8...
+Done!
+```
+
+Once the local network is running, the connection configuration file will be already in the correct place for your applications to connect to this network, so you can simply any application from this moment on to connect to your local network. Note that depending on the application, you may need to restart it so it uses the new connection information for your local network.
```

Comment:
> so you can simply any application
Looks like the word `run` was missed in the middle of this

Reply:
yes, thanks I'll correct that